charts/lgtm-distributed/Chart.yaml

```diff
@@ -16,7 +16,14 @@ sources:
   - https://github.com/grafana/mimir
   - https://github.com/grafana/tempo
 
+keywords:
+  - monitoring
+  - traces
+  - metrics
+  - logs
+
 annotations:
+  "artifacthub.io/license": AGPL-3.0-only
```

Comment:
This is not the correct license according to the license in this repo

Reply:
- fixed. https://github.com/grafana/helm-charts/pull/2811/commits/da737f8039b70bd436a870d64dec0b60cadfc10c